gensim/corpora/wikicorpus.py

```diff
@@ -255,6 +264,14 @@ def init_to_ignore_interrupt():
     signal.signal(signal.SIGINT, signal.SIG_IGN)
 
 
+def _process_article(args):
+    """Should not be called explicitly. use process_article instead"""
+    tokenizer_func, token_min_len, token_max_len, lower = args[-1]
+    args = args[:-1]
+    return process_article(args, tokenizer_func=tokenizer_func, token_min_len=token_min_len,
+                           token_max_len=token_max_len, lower=lower)
```

Comment:
I'd split those arguments into two lines, otherwise looks good 👍 

@menshikh-iv is working on cleaning up the code style in other parts of gensim, so I wouldn't worry about that. Let's just not introduce any new issues.